setup.py

```diff
@@ -4,8 +4,8 @@
 import sys
 
 ver_info = sys.version_info
-if ver_info < (3,5,4):
-    raise RuntimeError("canine requires at least python 3.5.4")
+if ver_info < (3,7,0):
```

Comment:
Is this too onerous a requirement? We can explicitly switch to ordered dicts if so.

Reply:
Agreed! Let's see how many alpha testers complain that it won't install because they don't have a new enough Python.